pytest_perf/runner.py

```diff
@@ -78,7 +79,18 @@ def __init__(self, extras=(), deps=(), control=None):
         spec = f'[{",".join(extras)}]' if extras else ''
         self.stack = contextlib.ExitStack()
         self.control_env = self._setup_env(upstream_url(spec, control), *deps)
-        self.experiment_env = self._setup_env(f'.{spec}', *deps)
+        target = os.environ.get("PYTEST_PERF_WHEEL_TARGET")
```

Comment:
Before this environment variable, this code was much more independent from its context. This environment variable entangles the implementation with the package in which it's contained and the context in which it's used. I'd prefer to see this class take a parameter and have the caller alter it. I'd like to explore who is the caller of BenchmarkRunner and is that perhaps a better place to solicit the alternate configurations. I'd also like to explore if an environment variable is the best way to alter the behavior or if, as a pytest plugin, it makes sense to solicit the config through pytest's settings.

Reply:
I gave it a try to use pytest CLI options. Please let me know if that works better.